framework/include/tinyalsa/tinyalsa.h

```diff
@@ -244,186 +244,205 @@ enum pcm_param {
 struct pcm;
 
 /**
-* @brief Opens a PCM for playback or recording.
-*
-* @param[in] The card that the pcm belongs to.
-* @param[in] The device that the pcm belongs to.
-* @param[in] flags Specify characteristics and functionality about the pcm.
-* @param[in] config The hardware and software parameters to open the PCM with.
-* @returns On success, A PCM structure returned. On failure, invalid PCM structure returned.
-* @since Tizen RT v1.1
-*/
+ * @brief Opens a PCM for playback or recording.
+ *
+ * @details @b #include <tinyalsa/tinyalsa.h>
+ * @param[in] The card that the pcm belongs to.
```

Comment:
You should write the variable name first and then you should add description of variable.
```
* @param[in] card    The card that the pcm belongs to.
* @param[in] device The device that the pcm belongs to.
* @param[in] flags   Specify characteristics and functionality about the pcm.
```

Reply:
modify